src/mc_cell_group.hpp

```diff
@@ -59,8 +60,36 @@ class mc_cell_group: public cell_group {
             ++source_gid;
         }
         EXPECTS(spike_sources_.size()==n_detectors);
+
```

Comment:
Yes it is from the model section.
This is a better spot, though we could argue that it still isn't the best spot. I am going to let Sam do the rationalization of this bit of code when he has the time!